action/envelope_test.go

```diff
@@ -0,0 +1,64 @@
+package action
```

Comment:
not a full test. Same for SealedEnvelope

Reply:
Repaired